tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -11,7 +11,7 @@
 from run_cumulus_task import run_cumulus_task
 from typing import List
 # instantiate Cumulus logger
-LOGGER = CumulusLogger()
+LOGGER = CumulusLogger(name="ORCA")
 
 EXCLUDE_FILE_TYPES_KEY = 'excludeFileTypes'
 CONFIG_COLLECTION_KEY = 'collection'
```

Comment:
Remove CONFIG_COLLECTION_KEY and COLLECTION_META_KEY since they are no longer used